src/LinuxCaptureService/MemoryWatchdogTest.cpp

```diff
@@ -45,13 +45,14 @@ TEST(MemoryWatchdog, ExtractRssInPagesFromProcPidStatReturnsNulloptWhenRssIsNotP
   EXPECT_FALSE(ExtractRssInPagesFromProcPidStat(kProcPidStat).has_value());
 }
 
-static void IncreaseRss(uint64_t amount_bytes) {
+static void IncreaseRss(uint64_t amount_words) {
   // The memory leak is intended: if the test is run again in the same process (e.g., with
```

Comment:
I guess this can still be considered a leak? Otherwise you could write something along the lines of (not sure how precise this is) "We intentionally keep the new memory allocation around until the process exits:".

Reply:
Yeah this is not on point anymore. I rephrased it a bit.